File: backend/src/main/java/com/backend/lane/domain/RegisterRequestDTO.java
package com.backend.lane.domain;

import lombok.Data;
import java.time.LocalDate;

@Data
public class RegisterRequestDTO {
 
    private String account_name;
    private String account_username;
    private String account_email;
    private String password; 
    private LocalDate account_dob;
    private Gender account_gender;
}

File: android/app/src/main/java/pt/iade/lane/data/models/RegisterRequestDTO.kt
package pt.iade.lane.data.models

import com.google.gson.annotations.SerializedName

data class RegisterRequestDTO(

    @SerializedName("account_name")
    val nome: String,

    @SerializedName("account_username")
    val username: String,

    @SerializedName("account_email")
    val email: String,

    @SerializedName("password")
    val passwordPura: String,

    @SerializedName("account_dob")
    val dataNascimento: String,

    @SerializedName("account_gender")
    val genero: String
)